scripts/generic-ci-tests.sh

```diff
@@ -80,8 +80,8 @@ case "$TEST_SUITE" in
         paver run_pylint -l $PYLINT_THRESHOLD > pylint.log || { cat pylint.log; EXIT=1; }
 
         mkdir -p reports
-        echo "Finding jshint violations and storing report..."
-        paver run_jshint -l $JSHINT_THRESHOLD > jshint.log || { cat jshint.log; EXIT=1; }
+        echo "Finding ESLint violations and storing report..."
+        paver run_eslint -l $ESLINT_THRESHOLD > eslint.log || { cat eslint.log; EXIT=1; }
```

Comment:
This will require a change to the jenkins job configuration for the edx-platform-quality jobs for master and PRs in order for the Quality Report (HTML report linked on the left sidebar) to replace the jshint log file with the eslint log file. Can you please put in a TE ticket? We will need to co-ordinate that change whenever this is merged in, keeping in mind how we will handle old branches and new.
It will also require a change to the code that runs the [edX Eng Dashboard](https://edx-eng-dashboard.herokuapp.com/edx). Please put in a TE ticket for that too.